src/Sarif/Writers/ResultLogJsonWriter.cs

```diff
@@ -245,10 +231,9 @@ public void OpenResults()
 
             _jsonWriter.WritePropertyName("results");
             _jsonWriter.WriteStartArray(); // Begin: results
-            _writeConditions = Conditions.ResultsInitialized;
+            _writeConditions |= Conditions.ResultsInitialized;
```

Comment:
>|= [](start = 29, length = 2)

Yikes!

Reply:
Test coverage hole?

---
In reply to: [255310688](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255310688) [](ancestors = 255310688)

Reply:
Yes there was. The introduction of Dispose handling has made sure we cover relevant code paths, so no new explicit test required. (This is, of course, how I found the bug, provoking relevant code paths due to writer changes...)

---
In reply to: [255310696](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255310696) [](ancestors = 255310696,255310688)